src/Compilers/CSharp/Portable/Symbols/Extensions/SourceExtensionImplementationMethodSymbol.cs

```diff
@@ -131,6 +133,19 @@ internal override int TryGetOverloadResolutionPriority()
             return UnderlyingMethod.TryGetOverloadResolutionPriority();
         }
 
+        public override string GetDocumentationCommentXml(CultureInfo? preferredCulture = null, bool expandIncludes = false, CancellationToken cancellationToken = default)
+        {
+            // Neither the culture nor the expandIncludes affect the XML for extension implementation methods.
+            string result = SourceDocumentationCommentUtils.GetAndCacheDocumentationComment(this, expandIncludes: false, ref lazyDocComment);
+
+#if DEBUG
+            string withIncludes = DocumentationCommentCompiler.GetDocumentationCommentXml(this, processIncludes: true, default);
```

Comment:
>DocumentationCommentCompiler.GetDocumentationCommentXml [](http://example.com/codeflow?start=34&length=55)

In order to make sure we compare apples to apples, I think ```SourceDocumentationCommentUtils.GetAndCacheDocumentationComment``` should be used here as well